frontend/packages/noobaa-storage-plugin/src/components/object-data-reduction-card/object-data-reduction-card.tsx

```diff
@@ -55,8 +55,8 @@ const DataReductionCard: React.FC<DashboardItemProps> = ({
     'loadError',
   ]);
 
-  const efficiency = getGaugeValue(efficiencyQueryResult);
-  const savings = getGaugeValue(savingsQueryResult);
+  const efficiency = Number(getGaugeValue(efficiencyQueryResult));
```

Comment:
Can we write another function in utils of extending the Gauge function(`getGaugeValue`) itself to say whether we want number or in string format?

Reply:
That can be done too but there are a lot of cleanup changes required in the noobaa plugin, which I will be doing in a different PR.

Reply:
We can and will but in a different PR. There are many cleanup changes required when we port our various cards to Resource Inventory Items and IMO would be better to do that then, we could generalize this function or might even come up with something completely different too.

Reply:
AHh ok, I agree